httpmw/ip_cognito_auth.go

```diff
@@ -181,7 +190,14 @@ func IpCognitoAuth(ipRange string, svc cognitoidentityprovideriface.CognitoIdent
 			return
 		}
 
-		username, ok := ch.Get(token)
+		model, ok := ch.Get(token)
+
+		if ok {
+			switch model := model.(type) {
+			case *CognitoUser:
+				user = model
+			}
+		}
```

Comment:
This can be simplified into a type assertion

```suggestion
		if model, ok := ch.Get(token); ok && model != nil {
		        if v, ok := model.(*CognitoUser); ok {
		                user = v
		        }
		}
```

Reply:
Updated.